migrations/000040_AddJwksUri.up.sql

```diff
@@ -0,0 +1,21 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+-- Allows us to periodically update the keys.
+ALTER TABLE HealthAuthority
+	ADD COLUMN jwks_uri VARCHAR(500);
```

Comment:
The maximum uniform resource locator (URL) length of 2,083 characters. Should this be `TEXT` instead?